presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcModule.java

```diff
@@ -43,6 +43,7 @@ public void configure(Binder binder)
         newOptionalBinder(binder, ConnectorAccessControl.class);
         newSetBinder(binder, Procedure.class);
         newSetBinder(binder, SessionPropertiesProvider.class).addBinding().to(BaseJdbcPropertiesProvider.class);
```

Comment:
Since I originally did this work someone recently added a `BaseJdbcPropertiesProvider` class (that only works for session properties not table properties). 

I can do the same if we want to do that.